recipe/run_test.sh

```diff
@@ -1,7 +1,8 @@
 #!/bin/bash
 
-export LINK=${CC}
+# export LINK=${CC}
 cd test
-sip -c . -b word.sbf word.sip
-python configure.py
-make
+sip-build
+# sip -c . -b word.sbf word.sip
+# python configure.py
+# make
```

Comment:
The same here, i.e. please remove all commented lines if they are not necessary anymore.